app/views/shared/_donatepod.html.haml

```diff
@@ -11,5 +11,4 @@
 
 - if AppConfig.bitcoin_donation_address
   Bitcoin:
-  %input{type: "text", id: "bitcoin_address", value: AppConfig.bitcoin_donation_address, readonly: true}
-
+  %input{type: "text", id: "bitcoin_address", class: "form-control", value: AppConfig.bitcoin_donation_address, readonly: true}
```

Comment:
- Avoid defining `class` in attributes hash for static class names
- Line is too long. [127/120]
- Files should end with a trailing newline